_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 14.0.0 Release"
+date: "2023-10-11 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 14.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
+
+## Java notes
```

Comment:
@danepitkin can you help filling those?

Reply:
```suggestion
## Java notes

Java 21 is enabled and validated in CI ([GH-37914](https://github.com/apache/arrow/issues/37914)).

The Gandiva module implemented a breaking change by moving Types.proto into a subfolder ([GH-37893](https://github.com/apache/arrow/issues/37893)).

`DefaultVectorComparators` added support for `LargeVarCharVector` and `LargeVarBinaryVector` ([GH-37886](https://github.com/apache/arrow/issues/37886))

A bug was fixed in `VectorAppender` to prevent resizing the data buffer twice when appending variable-length vectors ([GH-37829](https://github.com/apache/arrow/issues/37829)).


```

Reply:
Note: The Java 21 issue is not yet merged! If it doesn't make it in, we need to remove the issue.